engine/backend/common/src/main/java/com/torodb/backend/AbstractIdentifierConstraints.java

```diff
@@ -64,8 +64,16 @@ protected AbstractIdentifierConstraints(ImmutableSet<String> restrictedSchemaNam
         // No-Mongo types
         .put(FieldType.DATE, 'c') // [c]alendar
         .put(FieldType.TIME, 'm') // ti[m]e
+        .put(FieldType.JAVASCRIPT, 'j') // [j]avascript
+        .put(FieldType.JAVASCRIPT_WITH_SCOPE, 'w') // javascript [w]ith scope
+        .put(FieldType.MIN_KEY, 'k') //min [k]ey
+        .put(FieldType.MAX_KEY, 'a') //m[a]x key
+        .put(FieldType.UNDEFINED, 'u') //[u]ndefined
+        .put(FieldType.MONGO_REGEX, 'g') //Re[g]Ex
+        .put(FieldType.MONGO_DB_POINTER, 'p') //DB [p]ointer
+        .put(FieldType.DEPRECATED, 'z') //running out of available chars; z means "the last".
         .build());
-
+    //Letters left: f h o v
```

Comment:
Yes, these are the only letters left. I thought it would be useful for the next time we need to add a field